### ISSUE TITLE
Clarification on MD5 property

### ISSUE THREAD

**Issue by zonia3000** [ivoa-std/VOSpace#14]
In the `setNode` example (section 6.3.2) the response payload contains the following property URI: `ivo://ivoa.net/vospace/core#MD5`. It looks like a standard property, however it is not listed in the standard properties section.

Should we add it to the list? It seems that it is already used by CADC implementation. Also INAF implementation is going to use it.

Should we add other properties for other hashing algorithms that could be used to check file integrity (e.g. SHA256)?

**Comment by Zarquan** [comment]
I agree, we should add these. We would need to define the algorithm and parameters so that we get reproducible results on different platforms.

**Comment by Zarquan** [comment]
I agree we should add these. We should specify the algorithm and parameters for each type so that we get reproducible results on different platforms.

However, we should be careful to specify where they can be used. In the example you cite, in the `setNode` response, the `ivo://ivoa.net/vospace/core#MD5` and `ivo://ivoa.net/vospace/core#length` properties are applied to the node itself, which doesn't make sense.

It would be more accurate to specify a view, even if it is `ivo://ivoa/net/vospace/core#defaultview`, and apply the `length` and `MD5` properties to that.

    <vos:node uri="vos://cadc.nrc.ca!vospace/mydata1" xsi:type="vos:DataNode">
      <vos:properties>
        ....
      </vos:properties>
      <vos:accepts>
        <vos:view uri="ivo://ivoa.net/vospace/core#anyview"/>
      </vos:accepts>
      <vos:provides />
        <vos:view uri="ivo://ivoa.net/vospace/core#defaultview">
            <vos:property uri="ivo://ivoa.net/vospace/core#length" readOnly="true">44</vos:property>
            <vos:property uri="ivo://ivoa.net/vospace/core#MD5"    readOnly="true">12c13e49f2fd37e2c91bcca6acb78a08</vos:property>
        </vos:view>
      </vos:provides>
    </vos:node>
